.github/dependabot.yml

```diff
@@ -7,16 +7,22 @@ updates:
   open-pull-requests-limit: 10
 - package-ecosystem: npm
   directory: "/analysis/node-wrapper"
+  # Check the npm registry for updates at 3am UTC
   schedule:
     interval: daily
+    time: "03:00"
   open-pull-requests-limit: 10
 - package-ecosystem: gradle
   directory: "/analysis"
+  # Check gradle dependencies for updates at 3am UTC
   schedule:
     interval: daily
+    time: "03:00"
   open-pull-requests-limit: 10
 - package-ecosystem: npm
   directory: "/visualization"
+  # Check the npm registry for updates at 3am UTC
   schedule:
     interval: daily
+    time: "03:00"
```

Comment:
I suggest to change this to e.g., midnight.

Reply:
I think the problem is caused by something else. For example, dependabot force-pushes changed integrity checksums in the package-lock.json which leads to a new build every time: https://github.com/MaibornWolff/codecharta/pull/1414

Would changing the number of times we check if a PR is ready to merge solve the problems?

Reply:
The actual problem is, that some dependabot branches will never be merged because the number of retries is to low.
... Resolved